plugins/performance-lab/includes/site-health/audit-enqueued-assets/helper.php

```diff
@@ -293,6 +293,11 @@ function perflab_aea_get_path_from_resource_url( string $resource_url ): string
 		return '';
 	}
 
+	// Remove query string if present.
+	if ( false !== strpos( $resource_url, '?' ) ) {
+		$resource_url = substr( $resource_url, 0, strpos( $resource_url, '?' ) );
+	}
```

Comment:
```suggestion
	$resource_url = preg_replace( '/\?.*/', '', $resource_url );
```

Nevertheless, the `perflab_aea_get_path_from_resource_url()` function isn't being used now anymore, right?

Reply:
Yes its not being used anymore I just put comment about it here https://github.com/WordPress/performance/pull/2059#discussion_r2169768725.